iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift

```diff
@@ -11,7 +11,13 @@ import Frames
 import Checkout
 
 #if UITEST
-extension Factory {
+enum UITestFactory {
```

Comment:
Note related to this PR : Just a question why are all these enums and not structs ? examples are `enum Factory`, `enum ThemeDemo` etc. My option for enum would be to have similar values grouped as cases and not add static functions. Just wanted to understand this pattern.